packages/preview/lth-msc-cs/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "lth-msc-cs"
```

Comment:
If this package is not officially endorsed by the university, I will ask you to change the name to something less canonical. If it is an official package, can a member of the staff send an email (using their academic address) to `hello@typst.app` to say so, please?